### PR TITLE
Perform Centreon update via API

### PR DESCRIPTION
**Pull request by smau-centreon**
## Description

Perform Centreon update via API

## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [x] 22.04.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [ ] 22.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/update/update-centreon-platform.md [hunk 1]
```diff
@@ -33,7 +53,30 @@ Mettez à jour l'ensemble des composants :
...
 
-### Finalisation de la mise à jour
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+Videz le cache :
+
+```shell
+apt update && apt upgrade
+apt clean all
```
Comment (lpinsivy) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/update/update-centreon-platform.md:63]:
```suggestion
apt clean all
apt update
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/update/update-centreon-platform.md [hunk 2]
```diff
@@ -33,7 +53,30 @@ Mettez à jour l'ensemble des composants :
...
+```
+
+Mettez à jour l'ensemble des composants :
+
+```shell
+apt install centreon\*
```
Comment (lpinsivy) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/update/update-centreon-platform.md:69]:
```suggestion
apt upgrade centreon\*
```

File: versioned_docs/version-22.04/update/update-centreon-platform.md [hunk 3]
```diff
@@ -32,7 +52,30 @@ Then upgrade all the components with the following command:
...
+
+```shell
+apt update && apt upgrade
+apt clean all
+```
+
+Then upgrade all the components with the following command:
+
+```shell
+apt install centreon\*
```
Comment (lpinsivy) [inline versioned_docs/version-22.04/update/update-centreon-platform.md:68]:
```suggestion
apt upgrade centreon\*
```

File: versioned_docs/version-22.04/update/update-centreon-platform.md [hunk 4]
```diff
@@ -32,7 +52,30 @@ Then upgrade all the components with the following command:
...
 
-### Finalizing the update
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+Clean the cache:
+
+```shell
+apt update && apt upgrade
+apt clean all
```
Comment (lpinsivy) [inline versioned_docs/version-22.04/update/update-centreon-platform.md:62]:
```suggestion
apt clean all
apt update

```

**Review by lpinsivy — CHANGES_REQUESTED** [review]

File: versioned_docs/version-22.04/update/update-centreon-platform.md [hunk 5]
```diff
@@ -32,7 +52,31 @@ Then upgrade all the components with the following command:
...
+```
+
+</TabItem>
+</Tabs>
+
+> Now you can either perform the update:
+> - [Using the wizard](#finalize-the-update-using-wizard)
+> - [Using a dedicated API endpoint](#finalize-the-update-using-api-endpoint)
+
+#### Finalize the update using wizard
```
Comment (paul-oureib) [inline versioned_docs/version-22.04/update/update-centreon-platform.md:79]: #### Finalize the update using the wizard

File: versioned_docs/version-22.04/update/update-centreon-platform.md [hunk 6]
```diff
@@ -32,7 +52,31 @@ Then upgrade all the components with the following command:
...
-### Finalizing the update
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+Clean the cache:
+
+```shell
+apt clean all
+apt update
+
```
Comment (paul-oureib) [inline versioned_docs/version-22.04/update/update-centreon-platform.md:63]: line 63 is an empty, superfluous line within the code snippet

File: versioned_docs/version-22.04/update/update-centreon-platform.md [hunk 7]
```diff
@@ -70,6 +114,63 @@ this command:
 systemctl restart cbd centengine gorgoned
 ```
 
+You can now move to this [step](#update-extensions).
+
+#### Finalize the update using API endpoint
```
Comment (paul-oureib) [inline versioned_docs/version-22.04/update/update-centreon-platform.md:119]: #### Finalize the update using the API endpoint

File: versioned_docs/version-22.04/update/update-centreon-platform.md [hunk 8]
```diff
@@ -70,6 +114,63 @@ this command:
...
+  "security": {
+    "credentials": {
+      "login": "Admin",
+      "password": "xxxxx"
+    }
+  }
+}'
+```
+
+This is how the result should look like:
```
Comment (paul-oureib) [inline versioned_docs/version-22.04/update/update-centreon-platform.md:149]: This is what* the result should look like

File: versioned_docs/version-22.04/update/update-centreon-platform.md [hunk 9]
```diff
@@ -32,7 +52,31 @@ Then upgrade all the components with the following command:
...
+```shell
+apt upgrade centreon\*
+```
+
+</TabItem>
+</Tabs>
+
+> Now you can either perform the update:
+> - [Using the wizard](#finalize-the-update-using-wizard)
+> - [Using a dedicated API endpoint](#finalize-the-update-using-api-endpoint)
```
Comment (paul-oureib) [inline versioned_docs/version-22.04/update/update-centreon-platform.md:77]: Now you can either perform the update: 
- [Using the wizard](#finalize-the-update-using-the-wizard)
- [Using a dedicated API endpoint](#finalize-the-update-using-the-api-endpoint)

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/update/update-centreon-platform.md [hunk 10]
```diff
@@ -73,6 +116,64 @@ la commande suivante:
...
+  "security": {
+    "credentials": {
+      "login": "Admin",
+      "password": "xxxxx"
+    }
+  }
+}'
+```
+
+Voici ce à quoi le résultat ressemble :
```
Comment (paul-oureib) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/update/update-centreon-platform.md:152]: Voici à quoi ressemble le résultat :

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/update/update-centreon-platform.md [hunk 11]
```diff
@@ -73,6 +116,64 @@ la commande suivante:
...
 ```
 
+Vous pouvez maintenant passer à cette [étape](#mise-à-jour-des-extensions).
+
+#### Finalisation de la mise à jour avec l'API
+
+Connectez-vous au serveur Central via le terminal pour poursuivre le processus de
+mise à jour.
+
+Vous avez besoin d'un token d'authentification pour accéder au endpoint de l'API. Suivez cette procédure pour obtenir un numéro de token.
```
Comment (paul-oureib) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/update/update-centreon-platform.md:126]: Vous avez besoin d'un token d'authentification pour accéder à l'endpoint de l'API. Suivez cette procédure pour obtenir un token.

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by paul-oureib — APPROVED** [review]

**Pull request merged**